codegen/package.go

```diff
@@ -63,6 +63,8 @@ type PackageHelper struct {
 	moduleSearchPaths map[string][]string
 	// defaultDependencies is a dictionary of glob patterns for default dependencies
 	defaultDependencies map[string][]string
+	// clientinterfacesList is a list of interfaces to mock for a given client
+	clientInterfacesMock map[string][]string
```

Comment:
nit : comment -> its a map and not a list.

Reply:
Done